docs/configuration/http_filters/router_filter.rst

```diff
@@ -50,7 +50,7 @@ x-envoy-max-retries
 
 If a retry policy is in place, setting this header controls the number of retries that Envoy will
 perform on the request's behalf. If the header is not specified and there is no :ref:`route
-configuration <config_http_conn_man_route_table_route_retry>` Envoy will perform a single retry. A
+configuration <config_http_conn_man_route_table_route_retry>` Envoy will NOT attempt any retries. A
```

Comment:
@christian-posta In reading this, I think I realize what I originally meant. Basically, if someone sets x-envoy-retry-on, or has a route policy that defines what to retry on, the default is a single retry. So that basically implicitly turns on a single retry and this header is not needed. Can you clarify this in a way that makes sense to you given the new info?